sdcm/utils/operations_thread.py

```
@@ -126,9 +126,7 @@ def __init__(self, thread_params: ThreadParams, thread_name: str = ""):
         self.log = logging.getLogger(self.__class__.__name__)
         self.log.debug("Thread params: %s", thread_params)
         nemesis_seed = self.thread_params.db_cluster.params.get("nemesis_seed")
-        if isinstance(nemesis_seed, list):
-            nemesis_seed = nemesis_seed[0]
-        self.generator = random.Random(int(nemesis_seed)) if nemesis_seed is not None else random.Random()
+        self.generator = random.Random(nemesis_seed[0]) if nemesis_seed is not None else random.Random()
```

Ruff (0.15.15)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)